src/components/back-top/nz-back-top.component.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  Component,
+  ViewEncapsulation,
+  OnInit,
+  Input,
+  TemplateRef,
+  EventEmitter,
+  Output,
+  Renderer2,
+  OnDestroy,
+  ContentChild
+} from '@angular/core';
+
+import {
+  trigger,
+  style,
+  transition,
+  animate
+} from '@angular/animations';
+import { Subscription } from 'rxjs/Subscription';
+import { Observable } from 'rxjs/Rx';

```

Comment:
Should only import specific type as suggested in https://github.com/NG-ZORRO/ng-zorro-antd/pull/29